Dockerfile.baremetal

```diff
@@ -1,14 +1,18 @@
-#FROM python:2.7
 FROM fedora:31
 RUN dnf install -y libvirt-libs pip python && \
    yum clean all && \
    rm -rf /var/cache/yum
 
-
 RUN pip install boto3
 RUN pip install botocore
 
-COPY ./openshift-install /
+ENV VERSION="4.5.0-0.nightly-2020-05-13-060344"
```

Comment:
maybe we can  pass the version as a parameter to docker build and document it in README?

Reply:
This is a hack.
What we should do is get the binary in runtime from the release image (as they do in hive).
Yet, this should suffice for now.